File: MonorailSwift/Helper/Monorail_OC.m

#import "Monorail_OC.h"
#import <MonorailSwiftTools/MonorailSwiftTools-Swift.h>

@implementation Monorail_OC

+ (void) load {
    [Monorail_OC oc_enableLogger];
    [Monorail_OC oc_writeLog];
}

@end

File: MonorailSwift/Helper/Monorail_OC.swift
import Foundation
import MonorailSwift

public extension Monorail_OC {
    @objc static func oc_enableLogger() {
        Monorail.enableLogger()
    }
    
    @objc static func oc_writeLog() {
        Monorail.writeLog()
    }
}

